Doc/library/ftplib.rst

```diff
@@ -22,7 +22,7 @@ as mirroring other FTP servers.  It is also used by the module
 Here's a sample session using the :mod:`ftplib` module::
 
    >>> from ftplib import FTP
-   >>> ftp = FTP('ftp.debian.org')     # connect to host, default port
+   >>> ftp = FTP('ftp.us.debian.org')     # connect to host, default port
```

Comment:
Please remove 3 spaces before the comment, so that these `#` can be aligned.

Reply:
Addressed in 971d84f 
Sorry I didn't notice that :sweat_smile: Thanks for pointing it out